src/redux/reducers/congress.js

```diff
@@ -0,0 +1,35 @@
+import { handleActions, combineActions } from 'redux-actions';
+import { congressActions } from '../actions';
+
+const initialState = {
+  loading: false,
+  congressCandidates: [],
+};
+
+const congressReducer = handleActions(
+  {
+    [combineActions(
+      congressActions.applyForCongress.call,
+      congressActions.getCongressCandidates.call,
+    )]: (state) => ({
+      ...state,
+      loading: true,
+    }),
+    [combineActions(
+      congressActions.applyForCongress.success,
```

Comment:
We shouldn't yet set loading to false on congressActions.applyForCongress.success.
On success, we should dispatch congressActions.getCongressCandidates.call to update list of candidates, and success of this action will set loading to false.